a.js

```diff
@@ -0,0 +1,9 @@
+const humberger = document.querySelector('.humberger');
+const ulnav = document.querySelector('.ulnav');
+const logo = document.querySelector('.aa');
+
+humberger.addEventListener('click', () => {
+  logo.classList.toggle('none');
+  ulnav.classList.toggle('active');
+  humberger.classList.toggle('active');
+});
```

Comment:
Kindly add the functionality that will make the menu disappear when a menu item is clicked as stated by the previous reviewer. Currently, when I scroll up after clicking on a menu item, the mobile menu is still active. 

- The same functionality that closes the menu when the times(X) icon is clicked should also be applied when any of the menu list items are clicked.